modules/drivers/presto-jdbc/test/metabase/test/data/presto_jdbc.clj

```diff
@@ -176,3 +171,32 @@
 (defmethod sql.tx/add-fk-sql :presto-jdbc
   [_driver _dbdef _tabledef _fielddef]
   nil)
+
+(defmethod tx/dataset-already-loaded? :presto-jdbc
```

Comment:
What happens in the case of a partial load? Should we check that all tables are loaded? What if loading, as the last operation, added a secret `_<dataset>_checksum` table+value?

Reply:
Yeah this is still a known problem that I'm hoping to address at some point with a follow-on PR. This is also a problem if test data changes for one reason or another. I did experiment with that secret checksum idea a while back for Postgres so I think there's some promise there. 

I'm hoping this PR is just the first of several incremental PRs that improve test data loading a lot